src/RestApiClient.js

```diff
@@ -47,12 +47,16 @@ class RestApiClient {
    * Send a DELETE request to the given path URI.
    *
    * @param  {String} path
+   * @param  {FormData|Object} body
+   * @param  {Object} headers
```

Comment:
This also doesn't match the method signature, but is unused. Should we remove this line?